build-aux/m4/bitcoin_secp.m4

```diff
@@ -82,3 +82,19 @@ if test x"$has_valgrind" != x"yes"; then
   AC_CHECK_HEADER([valgrind/memcheck.h], [has_valgrind=yes; AC_DEFINE(HAVE_VALGRIND,1,[Define this symbol if valgrind is installed])])
 fi
 ])
+
+dnl SECP_TRY_APPEND_CFLAGS([flags], VAR)
```

Comment:
From my understanding, the square brackets `[...]` in a comment, that describes a new macro, means that parameter is optional. They do not mean quoting. Right?

Reply:
fixed